gitpod-io/website
# ✨ Whitepaper comp. refactor & New whitepaper comp.

## Description

- Refactored Whitepaper component
- Addition of New Component which provides options for Logos (SVGs) & Title, Description text along with Whitepaper component.

## Related Issue(s)



Fixes #3647

## How to test


- Open Preview & see following page:
    - `/customers`



<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3649"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
